crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1274,7 +1285,6 @@ pub async fn run(
     };
 
     cli_args.command = Some(command);
-    cli_args.cwd = Some(repo_root.as_path().to_owned());
```

Comment:
as far as I can tell, we don't actually use the cwd as repo root anywhere. can correct if this isn't true

Reply:
Is this a necessary change for this PR? `repo_root` looks like a bad name since it can get it's value from `cli_args.cwd`. This value will now no longer be made absolute if it is given as a relative value.

Reply:
Unfortunately it is necessary, although I can just save the old cwd instead. We don't want the repo root to be our cwd and in fact, we probably don't want to be in a repo in the first place for turbo clone. This is a pretty big departure from our other commands.